documentation/paths/claims-user.yaml

```diff
@@ -0,0 +1,31 @@
+get:
```

Comment:
It is different request, because GET /claims not only filter claims, but also request from auth-service info for claim owner's data. I do not need it when I search all user's claims. Also I will add pagination in the future, which is no needed for GET /claims.